kyuubi-common/src/test/scala/org/apache/kyuubi/operation/JDBCTests.scala

```diff
@@ -360,6 +361,14 @@ trait JDBCTests extends BasicJDBCTests {
     }
   }
 
+  test("kyuubi defined function - engine_name") {
```

Comment:
The test will fail in the MiniKube integration test due to the docker image issue. Please temporarily ignore it just like #938.